docs/cheatsheet.rst

```diff
@@ -114,10 +129,14 @@ The steps are as follows:
   hub issue create
   <enter text>
 
-2. Start a new feature with hubflow named: issue-<issue # you created in step 1>-<description of work>
+2. Start a new feature with hubflow named: :code:`issue-<issue # you created in step 1>-<description of work>`
 ::
   hub hf feature start issue-1-going-to-write-some-code
+
 3. Write code
+
+This is where the actual magic happens.
```

Comment:
I meant just the "This is where the actual magic happens" bit. If it's
already indented to the same level as "Write code", lets leave it as
is.